web/app/js/index.js

```diff
@@ -21,37 +22,45 @@ if (proxyPathMatch) {
 
 let controllerNs = appData.controllerNamespace || "conduit";
 
-let api = ApiHelpers(pathPrefix);
+const context = {
+  api: ApiHelpers(pathPrefix),
+  controllerNamespace: controllerNs,
```

Comment:
controllerNamespace does not appear to be propagated:
when I deploy the `emojivoto` demo app and look at the service mesh page for example, I see web pods showing up in the control plane table that aren't from the conduit namespace, indicating the controllerNamespace prop isn't getting set correctly.

Reply:
Ahhhhha, that explains it. Thank you. Lemme fix that up then =)